docs/pages/versions/unversioned/sdk/gl-view.md

```diff
@@ -239,3 +140,43 @@ Check [Reanimated documentation](https://docs.swmansion.com/react-native-reanima
 ## Remote Debugging & GLView
 
 This API does not function as intended with remote debugging enabled. The React Native debugger runs JavaScript on your computer (not the mobile device itself), and GLView requires synchronous native calls (which are not supported in Chrome).
```

Comment:
```suggestion
This API does not function as intended with remote debugging enabled. The React Native debugger runs JavaScript on your computer, not the mobile device. GLView requires synchronous native calls that are not supported in Chrome.
```

I think we can update this paragraph too with this PR. I do not think we need the text inside the brackets since it is part of the explanation. 

Sorry again about the unrelated change 😅